packages/constellation/src/components/ThemeProvider/ThemeProvider.tsx

```diff
@@ -1,26 +1,24 @@
-/* eslint-disable import/no-unresolved */
 import '../styles.css'
```

Comment:
`styles.css` is created at build time, which is why the eslint rule was disabled here.